### PR TITLE
DEV-5017: Upgrade nest to 7.1.2

### PR DESCRIPTION
**Pull request by gmenih341**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,36 +24,33 @@
...
   "dependencies": {
     "@bugsnag/js": "6.5.2",
-    "@bugsnag/plugin-express": "6.5.1",
-    "@nestjs/common": "6.11.11",
-    "@nestjs/core": "6.11.11",
-    "@nestjs/microservices": "6.11.11",
-    "@nestjs/platform-express": "6.11.11",
-    "@nestjs/testing": "6.11.11",
-    "@nestjs/websockets": "6.11.11",
+    "@bugsnag/plugin-express": "6.4.1",
```
Comment (rokerkony) [inline package.json:27]: why to shift bugsnag down?

File: package.json [hunk 2]
```diff
@@ -24,36 +24,33 @@
...
     "json-stringify-safe": "5.0.1",
-    "moment": "2.24.0",
+    "moment": "2.26.0",
     "reflect-metadata": "0.1.13",
-    "rxjs": "6.5.4",
-    "typescript": "3.8.3",
-    "uuid": "7.0.2"
+    "rxjs": "6.5.5",
+    "typescript": "3.7.2",
+    "uuid": "8.1.0"
```
Comment (rokerkony) [inline package.json:40]: https://github.com/uuidjs/uuid/blob/master/CHANGELOG.md#800-2020-04-29

it is weird that we had it set already for v8 :-/ hmm :-/ but looks good

File: package.json (hunk 1 above)
Comment (gmenih341) [reply package.json]: Reverted this change.. I tried to upgrade it, but there were many breaking changes that I didn't want to get into, so I downgraded it to the wrong version

Comment (rokerkony) [reply package.json]: we actually have a ticket for it

**Pull request merged**